src/writer/core.py

```
@@ -1283,11 +1283,17 @@ def run_blueprint_via_api(
         blueprint_id = payload.pop("blueprint_id", None)
         if not blueprint_id:
             raise ValueError("Missing blueprint_id in payload")
+        trigger_type = payload.pop("trigger_type", None)
+        if not blueprint_id:
+            raise ValueError("Missing trigger_type in payload")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)